src/core/pre-process.js

```diff
@@ -9,43 +9,30 @@
  *      want to be using a new module with your own profile
  */
 import { showError } from "./utils.js";
-import { sub } from "./pubsubhub.js";
 
 export const name = "core/pre-process";
 
-let doneResolver;
-export const done = new Promise(resolve => {
-  doneResolver = resolve;
-});
-
-sub(
-  "start-all",
-  async config => {
-    const result = [];
-    if (Array.isArray(config.preProcess)) {
-      const promises = config.preProcess
-        .filter(f => {
-          const isFunction = typeof f === "function";
-          if (!isFunction) {
-            const msg = "Every item in `preProcess` must be a JS function.";
-            showError(msg, name);
-          }
-          return isFunction;
-        })
-        .map(async f => {
-          try {
-            return await f(config, document);
-          } catch (err) {
-            const msg = `Function ${f.name} threw an error during \`preProcess\`.`;
-            const hint = "See developer console.";
-            showError(msg, name, { hint });
-            console.error(err);
-          }
-        });
-      const values = await Promise.all(promises);
-      result.push(...values);
-    }
-    doneResolver(result);
```

Comment:
Unless we had a reason to resolve with `result`...

Reply:
Probably not.